src/lv_misc/lv_anim.h

```diff
@@ -19,12 +19,20 @@ extern "C" {
 #include "../../../lv_conf.h"
 #endif
 
-#if LV_USE_ANIMATION
-
 #include <stdint.h>
 #include <stdbool.h>
 #include <string.h>
 
+/*Can be used to indicate if animations are enabled or disabled in a case*/
+enum {
+ LV_ANIM_OFF,
+ LV_ANIM_ON,
+};
+
+typedef uint8_t lv_anim_enable_t;
+
```

Comment:
This should be below the `TYPEDEFS` comment.

Reply:
I had to move it out of the `#if LV_USE_ANIMATION` because a lot of functions still need this typedef even if animations are disabled. We could add another `TYPEDEFS` comment outside of `#if LV_USE_ANIMATION` if we want.

Reply:
I've fixed it here: https://github.com/littlevgl/lvgl/pull/1106/commits/8173c427aae1227a893706407a6a86fe031871b8